include/algs/Stack.h

```diff
@@ -0,0 +1,39 @@
+#include "coord.h"
+#include "Tile.h"
+class Stack{
+    private: 
+        uint8_t mark;
+        coord stack[50];
+    public:
+        Stack(){
+            mark = 0;
+            for(int i = 0; i < 50; i++){
```

Comment:
dont use magic numbers please
```suggestion
            for(int i = 0; i < VARIABLE/CONSTANTS; i++){
```

Reply:
Okay, change made.